src/minErr.js

```diff
@@ -51,13 +51,19 @@ function minErr(module, ErrorConstructor) {
       return match;
     });
 
-    message += '\nhttp://errors.angularjs.org/"NG_VERSION_FULL"/' +
+    var url = 'http://errors.angularjs.org/"NG_VERSION_FULL"/' +
       (module ? module + '/' : '') + code;
 
     for (i = 0, paramPrefix = '?'; i < templateArgs.length; i++, paramPrefix = '&') {
-      message += paramPrefix + 'p' + i + '=' + encodeURIComponent(templateArgs[i]);
+      url += paramPrefix + 'p' + i + '=' + encodeURIComponent(templateArgs[i]);
+
+      if (url.length > minErrConfig.urlMaxLength) {
+        url = url.substr(0, minErrConfig.urlMaxLength - 3) + '...';
+        break;
```

Comment:
@gkalpak
 I feel it is confusing too, if we set urlMaxlength to 1000, then it will substr(0, 997) since it will add '...'
So the max length become 997 not 1000. 
Should I remove '...' ? but still need something to tell that this is not the full url.
Do you have any suggestion ?
Should I substr(0, 1000) then add '...' and handle this in the unit test by asserting it to equal 1003